miniapp/build.gradle

```diff
@@ -59,7 +59,11 @@ dependencies {
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$kotlin_coroutines"
-    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$kotlin_coroutines"
+    implementation ("org.jetbrains.kotlinx:kotlinx-coroutines-android") {
+        version {
+            strictly "$kotlin_coroutines"
```

Comment:
After `implementation` of `signatureverifier` I have got the following issue while launching miniapp, even though I didn't initialize signature verification SDK / any other change.

```
java.lang.VerifyError: Verifier rejected class com.rakuten.tech.mobile.miniapp.storage.FileWriter: java.lang.Object 
com.rakuten.tech.mobile.miniapp.storage.FileWriter.unzip(java.io.InputStream, java.lang.String, kotlin.coroutines.Continuation) failed to verify: java.lang.Object 
...
at com.rakuten.tech.mobile.miniapp.MiniApp$Companion$init$1.invoke(MiniApp.kt:179)
...
at com.rakuten.tech.mobile.miniapp.MiniAppDownloader.getStorage(Unknown Source:7)
...
com.rakuten.tech.mobile.miniapp.MiniAppDownloader.retrieveDownloadedVersionPath(MiniAppDownloader.kt:104)
...
```

I have realized that `signatureverifier` SDK is using following dependency e.g. [org.jetbrains.kotlinx:kotlinx-coroutines-android:1.4.1](https://github.com/rakutentech/android-signatureverifier/blob/8f54643a7855bbdbe47496c818f27f01e56118b3/signatureverifier/build.gradle#L52) which is causing the bug in MiniApp SDK. 

So, I made this change to use `$kotlin_coroutines` = `1.3.3` strictly.